tests/test_estimation.py

```diff
@@ -1145,7 +1145,7 @@ def test_crossval_for_aalen_add(self, data_pred2, data_pred1):
 
             expected = 0.90
             msg = "Expected min-mean c-index {:.2f} < {:.2f}"
-            assert np.mean(mean_scores) > expected, msg.format(expected, scores.mean())
+            assert np.mean(mean_scores) > expected, msg.format(expected, np.mean(scores))
```

Comment:
this change was a bug uncovered when this `assert` started failing. 

It's interesting the understand why this test started failing. Initially, for AAF, I used `self.predict_cumulative_hazard(X).iloc[-1].to_frame()` (no negative sign). This means that higher values would mean higher chances of dieing (as it's a hazard). But this test failed because the concordance was backwards - I would rank individuals who have a predicted small lifetimes as having the highest. So I added the negative sign and everything works. 

The problem is now in the interpretation of `predict` for use outside concordance index metrics. The user will get back negative values, which is kind of odd.